website/content/en/preview/reference/instance-types.md

```diff
@@ -3874,151 +3874,6 @@ below are the resources available with some assumptions and after the instance o
  |pods|737|
  |vpc.amazonaws.com/efa|1|
  |vpc.amazonaws.com/pod-eni|107|
-## c7g-flex Family
```

Comment:
any idea why these are getting removed?

Reply:
This would be happening due to the instance gen script. do you know if these instances are available in us-east-2